predictoor/pdr-faq.md

```diff
@@ -0,0 +1,140 @@
+
+Before we answer, please understand that to make decent money, it will take hundreds or thousands of submissions, guided by AI/ML models. So doing this manually would be tedious and error-prone. Therefore a bot is the most practical way. This is why OPF has made it easy to run a bot. Here are the Readme of both [Predictoor](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/predictoor.md) and [Trader](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/trader.md).  
+
+To the question "Is it possible to predict without a bot", the specific answer is "Yes". Here are the details
+- Somehow you need to submit an up/down prediction, with OCEAN stake, as a transaction to Sapphire chain. Here are the Possible ways to do such a transaction:
+  - (a) OPF-supplied Python bot
+  - (b) simple Python script
+  - (c) Use "Write Contract" in [Sapphire blockchain explorer](https://explorer.sapphire.oasis.io)
+  - (d) 3rd-party bot that decides to support this
+  - (e) dapp that decides to support this
+
+</details>
+
+### For Traders
+
+<details>
+
+<summary>How can I trust individual predictions?</summary>
+
+The most important metric to assess confidence in a prediction is "how much OCEAN have people *staked* on this prediction", both up and down. It's how much they're willing to lose if they're wrong. This metric is not gameable because OCEAN has scarcity. It's "put your money where your mouth is".
+
+</details>
+
+<details>
+
+<summary>What factors influence the accuracy of Predictoor? How accurate do you think the predictions will be over time?</summary>
+
+That's up to Predictoors to figure out! Certainly, historical price information has a huge influence. After that, there are many possibilities: other CEX and DEX data, on-chain data, social media data, GitHub data, news, weather, and more. The [Predictoor README](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/predictoor.md) gets you started, using historical price information.
+
+</details>
+
+### Price and Rewards
+
+<details>
+
+<summary>How is the pricing determined and will it continue to be 3 OCEAN?</summary>
+
+Pricing is 3 OCEAN for one feed for 24 hours. It will be like that for the near-term future. We can expect price changes and price structure over time, as the Predictoor core team learns what pricing makes sense.
+
+</details>
+
+<details>
+
+<summary>How do Predictoors benefit from sales? Is there a burn mechanism?</summary>
+
+20% of prediction feed sales go to OPF. A portion of that is passed on to the Ocean community, including burning.
+
+</details>
+
+<details>
+
+<summary>How Is Predictoor related to Data Farming?</summary>  
+
+Starting Nov 9 via "Predictoor Data Farming", there will be 37000 OCEAN (about $10K) and several $K worth of ROSE allocated weekly to Predictoor DF. This will be used to purchase prediction feeds, alongside purchases by traders, etc. 80% of prediction feed sales goes to predictoors themselves.
```

Comment:
Add at the beginning: `Predictoor is its own dapp, stack, and set of feeds. In addition, the Ocean Data Farming infrastructure is extended to incentivize for activities within Predictoor.`

`Nov 9` -> `Nov 9, 2023`. Why: ages better.

`there will be` -> `there is`. Why: ages better.